modules/tinymce/src/core/main/ts/init/InitIframe.ts

```diff
@@ -63,9 +63,10 @@ const getIframeHtml = (editor: Editor) => {
 };
 
 const createIframe = (editor: Editor, boxInfo: BoxInfo) => {
-  const iframeTitle = editor.translate('Rich Text Area');
+  const iframeTitle = Env.browser.isFirefox() ? Options.getIframeAriaText(editor) : 'Rich Text Area';
```

Comment:
Correct me if I am wrong. This line implies that `iframe_area_text` option is only be used on firefox.

Reply:
Yes, should've added it to the PR description. So this change will make screen reader announce the title attribute set to the iframe with the `iframe_aria_text` option, and fall back to the default `Rich Text Area. Press ALT-0 for help.` before this change, it wasn't announced at all when using Firefox, only `Rich Text Area frame`.  

But Chrome and Safari announce the `aria-label` of the body element, so setting to the body is valid. https://github.com/tinymce/tinymce/blob/e31f0559d45760036d5c2a48be4f7301b87927d4/modules/tinymce/src/core/main/ts/init/InitIframe.ts#L51